src/dcos_e2e/cluster.py

```diff
@@ -22,14 +22,14 @@
 @retry(
     exceptions=(subprocess.CalledProcessError),
     tries=60,
-    delay=1,
+    delay=5,
```

Comment:
Why?

Reply:
Arbitrary, it works. Might also work with 1 sec though.

Reply:
If this is a real fix for the issue let's take it out to a new PR / JIRA.